### DISCUSSION TITLE
How to quick find by commit hash?

### DISCUSSION THREAD

**Discussion by benxshen** [gitextensions/gitextensions#11473] [Q&A] [answered]
I remember that in the earlier versions, I could use Filter to enter the commit hash and find the commit record, but it seems that this function is gone after version 4?! How can I quickly find the commit record by commit hash?

It has been bothering me for a long time...

**Comment by RussKie** [comment, marked as answer]
[Navigate > Go to commit...] oe CTRL+SHIFT+G
